src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1410,7 +1410,7 @@ def from_json(cls, data: dict) -> 'ServiceDescription':
         return cls(spec=spec, events=events, **c_status)
 
     @staticmethod
-    def yaml_representer(dumper: 'yaml.SafeDumper', data: 'ServiceDescription') -> Any:
+    def yaml_representer(dumper: 'yaml.Dumper', data: 'ServiceDescription') -> yaml.Node:
```

Comment:
just curious, did mypy actually complain about the use of `Any` here, or did you just want to make the type more specific

Reply:
It complained. But after looking into it again, it's not merely a new version of mypy alone but a newer version of `types-PyYAML` is also needed to get the errors:
```
orchestrator/__init__.py:6: note: In module imported here:
orchestrator/_interface.py:1284: error: Argument 2 to "add_representer" has incompatible type "Callable[[SafeDumper, DaemonDescription], Any]"; expected "Callable[[Dumper, DaemonDescription], Node]"  [arg-type]
orchestrator/_interface.py:1417: error: Argument 2 to "add_representer" has incompatible type "Callable[[SafeDumper, ServiceDescription], Any]"; expected "Callable[[Dumper, ServiceDescription], Node]"  [arg-type]
orchestrator/__init__.py:3: note: In module imported here:
orchestrator/module.py:136: error: Argument 2 to "add_representer" has incompatible type "Callable[[SafeDumper, HostDetails], Any]"; expected "Callable[[Dumper, HostDetails], Node]"  [arg-type]

```